extension/src/popup/views/UnlockAccount/index.tsx

```diff
@@ -39,6 +40,8 @@ export const UnlockAccount = () => {
   const handleSubmit = async (values: FormValues) => {
     const { password } = values;
     await dispatch(confirmPassword(password));
+    // skip this location in history, we won't need to come back here after unlocking account
+    history.replace(destination);
```

Comment:
[replace](https://v5.reactrouter.com/web/api/history) replaces the current location in history stack, so replacing with next destination (which actually doesn't add since it's already there, but it prevents a new addition to the stack)